modules/tags.js

```diff
@@ -0,0 +1,235 @@
+      c.isTextBased() && !c.isThread() &&
+      !c.permissionOverwrites?.cache.get(msg.guild.id)?.deny?.has("ViewChannel")
+    ).map(c => c.toString()) : ["Here"];
+
+    const regex = /<@random ?\[(.*?)\]>/gm;
+    if (regex.test(response)) {
+      console.log("true");
+      const replace = (/** @type {string} */ str) => u.rand(str.replace(regex, '$1').split('|'));
+      console.log(response.replace(regex, replace));
+      response = response.replace(regex, replace);
+    }
+    response = response
+      .replace(/<@author>/ig, msg.author.toString())
+      .replace(/<@authorname>/ig, msg.member?.displayName || msg.author.username)
+      .replace(/<@channel>/ig, msg.channel.toString())
+      .replace(/<@randomchannel>/, u.rand(randomChannels) ?? msg.channel.toString());
+    if ((/(<@target>)|(<@targetname>)/ig).test(response)) {
+      if (!msg.guild) target ??= msg.client.user;
+      if (!target) return msg.reply("You need to `@mention` a user with that command!").then(u.clean);
+      response = response.replace(/<@target>/ig, target.toString())
+        .replace(/<@targetname>/ig, target.displayName);
+    }
+  }
+  msg.channel.send({
+    content: response ?? undefined,
+    files: tag.attachment ? [new u.Attachment(`./media/tags/${tag._id}`).setName(tag.attachment)] : [],
+    allowedMentions: { users: target ? [target.id, msg.author.id] : [msg.author.id] }
+  });
+}
+
+/** @param {Augur.GuildInteraction<"CommandSlash">} int */
+async function slashTagCreate(int) {
+  // Get and validate input
+  const name = int.options.getString('name', true).toLowerCase().replace("\n", " ");
+  const content = int.options.getString('content');
+  const attachment = int.options.getAttachment('attachment');
+  if (findTag(name)) return int.editReply(`Looks like that tag already exists. Try </tag modify:${u.sf.commands.slashTag}> or </tag delete:${u.sf.commands.slashTag}> instead.`);
+  if (!content && !attachment) return int.editReply("I either need content or a file.");
+  // Create the tag
+  const command = await u.db.tags.manageTag({
+    tag: name,
+    response: content ?? undefined,
+    attachment: attachment?.name
+  });
+  if (!command) return int.editReply("I wasn't able to save that. Please try again later or with a different name.");
+  if (attachment) saveAttachment(attachment, command);
+
+  tags.set(command.tag, command);
+  const embed = u.embed({ author: int.member })
+    .setTitle("Tag created")
+    .setDescription(`${int.member} added the tag \`${name}\``);
+  if (command.response) embed.addFields({ name: "Response", value: command.response });
+  try {
+    // report the tag creation
+    int.client.getTextChannel(u.sf.channels.team)?.send({ embeds: [embed], files: attachment ? [attachment] : [] });
+    int.editReply({ content: "Tag created!", embeds: [embed.setDescription(`Try it out with \`!${name}\``)], files: attachment ? [attachment] : [] });
```

Comment:
```suggestion
    int.editReply({ content: "Tag created!", embeds: [embed.setDescription(`Try it out with \`${config.prefix}${name}\``)], files: attachment ? [attachment] : [] });
```

Not a big deal but just in case. I'd also import config earlier, of course.